packages/base/Dropdown/src/Dropdown/use-content-classes.ts

```diff
@@ -0,0 +1,46 @@
+import { twJoin } from '@toptal/picasso-tailwind-merge'
+import React from 'react'
+
+const useContentClasses = (contentType: 'scroll' | 'visible') => {
```

Comment:
Maybe to name the variable `contentOverflow` to make it more readable (the variable that is passed to function in `packages/base/Dropdown/src/Dropdown/Dropdown.tsx` is also called `contentOverflow`)?

```suggestion
const useContentClasses = (contentOverflow: 'scroll' | 'visible') => {
```